exonum/src/blockchain/schema.rs

```diff
@@ -331,13 +338,22 @@ where
     fn next_height(&self) -> Height {
         Height(self.block_hashes_by_height().len())
     }
+
+    /// Count txs in pool
+    #[cfg_attr(feature = "cargo-clippy", allow(let_and_return))]
+    pub fn pool_len(&self) -> usize {
+        let pool = self.transactions_pool();
+        let count = pool.iter().count();
+        count
+    }
 }
 
 impl<'a> Schema<&'a mut Fork> {
     /// Mutable reference to the [`transactions`][1] index.
     ///
     /// [1]: struct.Schema.html#method.transactions
-    pub(crate) fn transactions_mut(&mut self) -> MapIndex<&mut Fork, Hash, RawMessage> {
+    #[doc(hidden)]
+    pub fn transactions_mut(&mut self) -> MapIndex<&mut Fork, Hash, RawMessage> {
```

Comment:
What's the reasoning for reverting `pub(crate)` visibility? I seriously doubt that there is a legitimate purpose for external apps to modify this index.

Reply:
Testkit and sandbox need access to mempool. For now user can create index with the same name as core tables, so we decided to make clever workaround later, after this problem will be fixed. 
Btw this index is hide and undocumented, so user shouldn't touch it.